avaframe/runScripts/runProbAnaCom1DFA.py

```diff
@@ -79,7 +79,7 @@
         log.warning('No files found for configuration: %s' % probConf)
```

Comment:
line 65, you also remove the work directory, is this wanted?

Reply:
yes, because otherwise it throws an error if a workDir is already existing, as it first calls initialiseDirs in com1DFAmain:
 in initialiseDirs:
if workDir.is_dir():
        message = 'Work directory %s already exists - delete first!' % (workDir)
        log.error(message)
        raise AssertionError(message)